src/rust/engine/process_execution/src/local.rs

```diff
@@ -878,9 +878,8 @@ pub fn setup_run_sh_script(
   let full_script = format!(
     "#!/bin/bash
```

Comment:
Since the script now requires env it might be a good time to switch to `#!/usr/bin/env bash` but I'm not sure which is more likely to be in a non-standard location on a host, `/bin/bash` or `/usr/bin/env`.

Reply:
`/usr/bin/env bash` is idiomatic for a reason, so presumably conventional wisdom believes that `/usr/bin/env` is more likely to be present?

Reply:
And I mean this only affects debugging anyway, so if you're on a non-standard system, you can always edit `__run.sh` to right it.